src/code_80057C60.c

```diff
@@ -4993,7 +4993,7 @@ void func_8006538C(Player* player, s8 arg1, s16 arg2, s8 arg3) {
         spAC[0] = 0;
         spAC[1] = player->unk_048[arg3];
         spAC[2] = 0;
-        if ((player->effects & STAR_EFFECT) && (((s32) gCourseTimer - D_8018D930[arg1]) < 9)) {
+        if ((player->effects & STAR_EFFECT) && (((s32) gCourseTimer - gPlayerStarEffectStartTime[arg1]) < 9)) {
```

Comment:
Maybe replace the constant 9 by STAR_DURATION -1

Reply:
Changed here [17a39a7](https://github.com/n64decomp/mk64/pull/720/commits/17a39a77336a0b697e7d1928316680cbcb68a0ce)